packages/front-end/components/Settings/Team/SingleRoleSelector.tsx

```diff
@@ -23,12 +22,10 @@ export default function SingleRoleSelector({
   includeAdminRole?: boolean;
   disabled?: boolean;
 }) {
-  const growthbook = useGrowthBook();
   const { roles, hasCommercialFeature } = useUser();
   const hasFeature = hasCommercialFeature("advanced-permissions");
-  const isNoAccessRoleEnabled =
-    growthbook?.isOn("no-access-role-type") &&
-    hasCommercialFeature("no-access-role");
+
+  const isNoAccessRoleEnabled = hasCommercialFeature("no-access-role");
 
   let roleOptions = [...roles];
 
```

Comment:
If we add additional roles that are only available for certain plans, or if we add support for custom roles, we should refactor the `getRoles` call to handle that logic on the backend. But for now, I think it's overkill for this.